src/main/java/com/stripe/net/ApiRequest.java

```diff
@@ -1,20 +1,49 @@
 package com.stripe.net;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Map;
 import lombok.Getter;
 
 @Getter
 public class ApiRequest extends BaseApiRequest {
   private Map<String, Object> params;
 
+  private ApiRequest(
+      BaseAddress baseAddress,
+      ApiResource.RequestMethod method,
+      String path,
+      RequestOptions options,
+      ApiMode apiMode,
+      List<String> usage,
+      Map<String, Object> params) {
+    super(baseAddress, method, path, options, apiMode, usage);
+    this.params = params;
+  }
+
   public ApiRequest(
       BaseAddress baseAddress,
       ApiResource.RequestMethod method,
       String path,
       Map<String, Object> params,
       RequestOptions options,
       ApiMode apiMode) {
-    super(baseAddress, method, path, options, apiMode);
-    this.params = params;
+    this(baseAddress, method, path, options, apiMode, null, params);
+  }
+
+  public ApiRequest withAddedUsage(String usage) {
```

Comment:
```suggestion
  public ApiRequest addUsage(String usage) {
```
? 

We use `add` prefix for collections in param builders.

Reply:
Added in [withAddedUsage -> addUsage](https://github.com/stripe/stripe-java/pull/1721/commits/90cf4b9005cf73532fe09693a4cfa94048c03c98)